### DISCUSSION TITLE
Read-only application cannot POST.

### DISCUSSION THREAD

**Discussion by hwalker928** [tweepy/tweepy#1463] [Questions] [answered]
Hey, I'm trying to create a tweet, the authentication is OK, but I get this error:

tweepy.error.TweepError: Read-only application cannot POST.

Thanks!

**Comment by Harmon758** [comment, marked as answer]
You need to change your app's permission level if you want it to have write access.
See https://developer.twitter.com/en/docs/apps/app-permissions.
